js/libs/keycloak-admin-client/src/defs/effectiveMessageBundleRepresentation.ts

```diff
@@ -0,0 +1,8 @@
+export default interface EffectiveMessageBundleRepresentation {
+  theme?: string;
+  themeType?: string;
+  local?: string;
```

Comment:
Not sure, but I think this should be `locale`.
```suggestion
  locale?: string;
```